packages/edit-site/src/components/editor/index.js

```diff
@@ -40,6 +45,7 @@ import BlockEditor from '../block-editor';
 import KeyboardShortcuts from '../keyboard-shortcuts';
 import GlobalStylesProvider from './global-styles-provider';
 import NavigationSidebar from '../navigation-sidebar';
+import useEditorStyles from '../../../../block-editor/src/components/editor-styles';
```

Comment:
Is this correct? Shouldn't it be:
```suggestion
import { __unstableUseEditorStyles as useEditorStyles } from '@wordpress/block-editor';
```
?

Reply:
Yes, thanks :) Sometimes VSCode wrongly auto imports